.gitignore

```diff
@@ -53,3 +53,4 @@ bilder.pyc
 bild.log
 
 bild_output.txt
+/bin/
```

Comment:
:bulb: consider setting this in **.git/info/exclude** instead, since the `bin` directory is not created by this project's build.